src/desktop/apps/viewing-room/server.tsx

```diff
@@ -0,0 +1,58 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { stitch } from "@artsy/stitch"
+import { routes as viewingRoomRoutes } from "reaction/Apps/ViewingRoom/routes"
+import React from "react"
+import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
+import express, { Request, Response, NextFunction } from "express"
+import { skipIfClientSideRoutingEnabled } from "desktop/components/split_test/skipIfClientSideRoutingEnabled"
+import adminOnly from "desktop/lib/admin_only"
+
+export const app = express()
+
+app.get(
+  "/viewing-room*",
+  skipIfClientSideRoutingEnabled,
```

Comment:
@dzucconi - this is the middleware we use to opt new pages into the new app shell